### DISCUSSION TITLE
how to register a new augmentation method and specific in config yml?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [facebookresearch/detectron2#3297] [Q&A]
Hi, is there existed GridMask augmentation in d2? How should I register it if there isn't one, and specify it in config yml?

**Comment by ppwwyyxx** [comment]
The tutorial has explained how to use new augmentation: https://detectron2.readthedocs.io/en/latest/tutorials/data_loading.html

We do not support defining new augmentation in yaml file, though this is support if using python config, e.g. https://github.com/facebookresearch/detectron2/blob/bb8b293c456c43b1745b9a49725eeed3bb965dc6/configs/new_baselines/mask_rcnn_R_50_FPN_100ep_LSJ.py#L43-L50

**Comment by lucasjinreal** [comment]
thanks, I solved this.
